pkg/oc/cli/cmd/extract.go

```diff
@@ -38,6 +41,9 @@ var (
 	  # extract the config map "nginx" to the /tmp directory
 	  %[1]s extract configmap/nginx --to=/tmp
 
+		# extract the config map "nginx" to STDOUT
+	  %[1]s extract configmap/nginx --to=-
```

Comment:
this seems weird, can't we just stream to stdout by default when `--to` is not set?

Reply:
We could, would you expect that?  Extract with STDOUT is more of a viewing action (like `kubectl view`), while extract to file is more "normal".

Reply:
@smarterclayton how about `--to-stdout` ?

Reply:
We use '-' pretty consistently.  Hrm... I guess I could do no arg.

Reply:
Hrm, no-arg today means "extract to current directory".  So if we change the behavior of no arg it's not backwards compatible.

Given that we use '-' in other places it's a pretty reasonable convention to continue.  Do we have any examples of --to-stdout or similar?

Reply:
Per our conversation, and given that conventions do exist to have `-` mean "output to stdin/stdout", I think it is reasonable to have `-` mean stdout in this case.